cypress/e2e/po/pages/explorer/projects-namespaces.po.ts

```diff
@@ -3,6 +3,30 @@ import LabeledInputPo from '@/cypress/e2e/po/components/labeled-input.po';
 import ResourceListMastheadPo from '@/cypress/e2e/po/components/ResourceList/resource-list-masthead.po';
 import NameNsDescription from '@/cypress/e2e/po/components/name-ns-description.po';
 import AsyncButtonPo from '@/cypress/e2e/po/components/async-button.po';
+import TabbedPo from '@/cypress/e2e/po/components/tabbed.po';
+import SortableTablePo from '@/cypress/e2e/po/components/sortable-table.po';
+
+class ProjectNamespaceDetailPagePo extends PagePo {
+  private static createPath(clusterId: string, id: string) {
+    return `/c/${ clusterId }/explorer/namespace/${ id }`;
+  }
+
+  goTo(clusterId: string, id: string): Cypress.Chainable<Cypress.AUTWindow> {
+    return super.goTo(ProjectNamespaceDetailPagePo.createPath(clusterId, id));
+  }
+
+  constructor(clusterId: string, id: string) {
+    super(ProjectNamespaceDetailPagePo.createPath(clusterId, id));
+  }
+
+  clickComplianceTab() {
```

Comment:
these two are specific to kw

Reply:
I've created a new class in kubewarden.utils called `KubewardenResourceComplianceReportPagePo` which is the same for any resource, afaik, and then imported it in `projectNamespaceDetail` to be exposed as `kubewardenComplianceReport`, which will access the methods.

I think the context is correct, to keep it accessible in projectsNamespaces, but the "ownership" I think you are right. What do you think?

Reply:
The aim is to keep kubewarden specific code out of generic code. 

I might be wrong, but if the compliance tab is an kw specific additional tab in the detail page of a project. so suggest the hierarchy as

```
KubewardenResourceComplianceReportTabPo extends ComponentPo {
  complianceSortableTable() {
    return new SortableTablePo(this.self().get('#policy-report-tab .sortable-table'));
  }
}

KubewardenProjectNamespaceDetailPagePo extends ProjectNamespaceDetailPagePo {
  clickComplianceTab() {
    return new TabbedPo().clickTabWithSelector('[data-testid="btn-policy-report-tab"]');
  }

  complianceTab() {
     return new KubewardenResourceComplianceReportTabPo(this.self())
  }
}
```

So there's a KW specific project namespace detail page, inheriting all the root page functionality, exposing new KW specific functionality